core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingComponent.java

```diff
@@ -970,6 +971,9 @@ public boolean execute() throws Exception {
       log.info( "Report execution interrupted." );
     } catch ( Exception e ) {
       log.error( Messages.getInstance().getString( "ReportPlugin.executionFailed" ), e ); //$NON-NLS-1$
+      if ( e instanceof ReportProcessingException ) {
```

Comment:
Why not a specific catch clause for ReportProcessingException?

Reply:
I could specify a catch clause for ReportProcessingException before the Exception's catch clause. 
But I would need to repeat line 973 inside the new catch.  https://github.com/pentaho/pentaho-platform-plugin-reporting/blob/b59e2f373e06b65e73614cd86e220e01633062be/core/src/main/java/org/pentaho/reporting/platform/plugin/SimpleReportingComponent.java#L973

Please tell me what version do you prefer, @pamval ?